scene/resources/curve.cpp

```diff
@@ -1414,7 +1414,7 @@ Vector3 Curve3D::interpolate_baked_up_vector(float p_offset, bool p_apply_tilt)
 	int count = baked_up_vector_cache.size();
 	if (count == 0) {
 		ERR_EXPLAIN("No up vectors in Curve3D");
-		ERR_FAIL_COND_V(count == 0, Vector3(0, 1, 0));
+		ERR_FAIL_COND_V(true, Vector3(0, 1, 0));
```

Comment:
Seems like you missed this occurrence that can be changed to `ERR_FAIL_V` too. Otherwise the rest is good.

Reply:
Maybe add a dot to the error too as done for other occurrences.

Reply:
This has been fixed, by the way.